src/libs/routes/authmiddleware.ts

```diff
@@ -0,0 +1,23 @@
+import * as jwt from 'jsonwebtoken';
+import hasPermission from '../permission' 
+
+export default (module , permissionType) =>(req , res, next) => {
+    try {
+        console.log('module and permission is ', module , permissionType);
+        console.log('header', req.header('authorization'));
+        const token = req.header('authorization');
+        const decode = jwt.verify(token, 'qwertyuiopasdfghjklzxcvbnm123456');
```

Comment:
Put the secret_key in .env file and import through config file same like PORT, NODE_ENV